extensions/mssql/src/main.ts

```diff
@@ -87,6 +87,30 @@ export async function activate(context: vscode.ExtensionContext): Promise<IExten
 		}
 	});
 
+	vscode.commands.registerCommand(Constants.cmdObjectExplorerEnableGroupBySchemaCommand, async () => {
+		await vscode.workspace.getConfiguration().update(Constants.configObjectExplorerGroupBySchemaFlagName, true, true);
+	});
+
+	vscode.commands.registerCommand(Constants.cmdObjectExplorerDisableGroupBySchemaCommand, async () => {
+		await vscode.workspace.getConfiguration().update(Constants.configObjectExplorerGroupBySchemaFlagName, false, true);
+	});
+
+	vscode.workspace.onDidChangeConfiguration(async e => {
+		if (e.affectsConfiguration(Constants.configObjectExplorerGroupBySchemaFlagName)) {
+			TelemetryReporter.sendTelemetryEvent('objectExplorerGroupBySchemaChanged', {
```

Comment:
It would be better to use an action event for this - aggregating and persisting custom events like this is much more difficult.